src/service/tag-service.js

```diff
@@ -0,0 +1,22 @@
+import TagRepogitory from '../repository/tag-repository.js';
+
+class TagService {
+  async getTags(page, limit, order, search) {
+    const where = {
+      name: { contains: search, mode: 'insensitive' },
+    };
```

Comment:
```suggestion
    const where = {};
    if (search) {
        where.name = { contains: search, mode: 'insensitive' };
    }
```
`search` 파라미터가 있을 때만 where 객체에 조건을 추가하는 것도 좋을 것 같습니다!